graylog2-web-interface/src/components/common/EntityList.md

```diff
@@ -37,7 +37,7 @@ const items = entities.map(entity => {
                     actions={[
                       <Button key={1} bsStyle="info">Edit</Button>,
                       <DropdownButton key={2} id="more-dropdown" title="More" pullRight>
-                        <MenuItem variant="danger">Delete</MenuItem>
+                        <MenuItem>Edit</MenuItem>
```

Comment:
I think we have to use here  delete menu item

Reply:
Thanks!